test/smoke/process-usage/src/test/fixtures.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+
+const timewindow = require('abacus-timewindow');
+
+const { times } = require('underscore');
+
+const { fixturesCfg } = require('./env-config');
+// The scaling factor of each time window for creating the date string
+// [Second, Minute, Hour, Day, Month]
+const _slack = () =>
```

Comment:
I think it would be better if slack and windows functions are moved to `timewindow` module?

Reply:
I am not sure for that. Could they be used in other modules with this implementation? It seems as functionality that might be used when we need to create report that we expect to get from Abacus.

Reply:
When we apply the solution with subtracting reports these utility functions will become redundant in smoke test